src/lib/requests/getForecastData.ts

```diff
@@ -51,9 +51,22 @@ export const getForecastData = async (
 ): Promise<ForecastDataType[] | undefined> => {
   if (!treeId) return
 
-  const REQUEST_URL = `${SUPABASE_PASSTHROUGH_API_URL}/${TABLE_NAME}?${TREE_ID_COLUMN_NAME}=eq.${treeId}&${TYPE_ID_COLUMN_NAME}=eq.${TYPE_ID_FOR_AVERAGE}&${TIMESTAMP_COLUMN}=gte.${TODAY}&order=${TIMESTAMP_COLUMN}&limit=${FORECAST_MAX_ROWS}&offset=0`
+  const REQUEST_URL = `${SUPABASE_PASSTHROUGH_API_URL}/${TABLE_NAME}`
 
-  const response = await fetch(REQUEST_URL, REQUEST_OPTIONS)
+  const REQUEST_PARAMS = new URLSearchParams({
```

Comment:
What I meant was to use the object as request body but this works great as well! 👍